app/enums/JourneyType/JourneyType.scala

```diff
@@ -18,7 +18,9 @@ package enums.JourneyType
 
 import enums.IncomeSourceJourney.IncomeSourceType
 
-case class JourneyType(operation: Operation, businessType: IncomeSourceType) {
+sealed trait JourneyType
+
+case class IncomeSources(operation: Operation, businessType: IncomeSourceType) extends JourneyType {
```

Comment:
this naming convention would create confusion, if you planning to generalise this
you can use something like: IncomeSourceJourneyType or similar to preserver meaning